src/main/res/values/strings.xml

```diff
@@ -828,7 +828,7 @@
 	<string name="pref_behaviour_fling_comment_right_title">Fling right</string>
 
 	<!-- 2016-07-17 -->
-	<string name="comment_header_search_thread_title">You are currently viewing search results for a comment thread.</string>
+	<string name="comment_header_search_thread_title">Viewing search results from a comment thread.</string>
```

Comment:
The original "for" made more sense

Reply:
Why? It isn't on part of, it is from a source.
"Viewing comment thread results" isn't that good, and neither is "in".(?)

Reply:
On re-reading I agree with you, let's make this "from" then. However I do prefer the "You are currently" in the original, it's clearer and friendlier :)